tests/baseline/cases/staged-simple-ami-bc.yml

```diff
@@ -21,9 +21,11 @@ items:
     - dest1
     ena_support: true
     image_id: null
+    marketplace_entity_type: null
```

Comment:
Ideally there should be some tests having non-null values for the new fields.  Otherwise it has never been tested that the fields can store any value other than None.

Reply:
ok, I will populate the new fields of new metadata in the test files.

Reply:
It seems like there are some code changes needed before this is possible.  Added a comment above on `_from_data` method.